Braintree.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,14 @@
+{
+  "pins" : [
```

Comment:
qq: Do we need to keep this or should we `.gitignore`?

Reply:
I was wondering on this and open to input. Wasn't sure of the best way to include NXO into the demo app, since the SDK frameworks are added to the demo app individually, but NXO wouldn't be included in that (since it's not embedded within the `BraintreePayPalNativeCheckout` module). Instead, it's resolved by the linker and would need to be retrieved (ie what SPM and CocoaPods are doing). I thought adding a package dependency into our demo app would be the most straightforward, but yeah, definitely open to suggestion if there are better ways.

Reply:
Yeah I agree. This file is generated by Xcode though yeah? I think it's similar to a `package-lock.json` file in node.js.

Reply:
Yeah i'm down to keep it then. We can always remove if it affects a merchant integration (it probably won't).